<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AGRCGeocodingToolboxDialogBase</class>
 <widget class="QDialog" name="AGRCGeocodingToolboxDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AGRC Geocoding Toolbox</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>341</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="apiKeyEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>24</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>API Key (obtain via https://api.mapserv.utah.gov)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>74</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address Table</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressTableEdit">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>94</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addressTableButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>54</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="idFieldComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>134</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Field</string>
   </property>
  </widget>
  <widget class="QComboBox" name="addressFieldComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>174</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address Field</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>206</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Zone Field</string>
   </property>
  </widget>
  <widget class="QComboBox" name="zoneFieldComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>246</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Locator</string>
   </property>
  </widget>
  <widget class="QComboBox" name="locatorComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputFolderEdit">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>344</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>327</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputFolderButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>54</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="spatialRefComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>286</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Spatial Reference</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>apiKeyEdit</tabstop>
  <tabstop>addressTableEdit</tabstop>
  <tabstop>addressTableButton</tabstop>
  <tabstop>idFieldComboBox</tabstop>
  <tabstop>addressFieldComboBox</tabstop>
  <tabstop>zoneFieldComboBox</tabstop>
  <tabstop>locatorComboBox</tabstop>
  <tabstop>spatialRefComboBox</tabstop>
  <tabstop>outputFolderEdit</tabstop>
  <tabstop>outputFolderButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>AGRCGeocodingToolboxDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>AGRCGeocodingToolboxDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
